src/github/github.ts

```diff
@@ -19,7 +19,11 @@ export const fetchWorkflowResults = async (
   maxTry = 10
 ): Promise<WorkflowResults> => {
   const token = core.getInput('GITHUB_TOKEN') || process.env.GITHUB_TOKEN // read environment variable for testing
-  const octokit = new Octokit({ auth: token })
+  const serverUrl = process.env.GITHUB_API_URL || 'https://api.github.com'
+  const octokit = new Octokit({
+    baseUrl: serverUrl,
+    auth: token
+  })
```

Comment:
I want to make `serverUrl` inline because it is used only once.

```suggestion
  const octokit = new Octokit({
    baseUrl: process.env.GITHUB_API_URL || 'https://api.github.com',
    auth: token
  })
```

Reply:
@paper2 
Change the base branch and run the npm run all succsesfuly